src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +19,12 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        return movieDao.get(id).orElseThrow(() ->
+                new RuntimeException("Movie not found by id: " + id));
```

Comment:
Consider using a more specific exception type instead of `RuntimeException` for better clarity and error handling. You could define a custom exception or use an existing one that better describes the error context.